_slides/06-horvatmatthew.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+The Flying Elbow Drop!  OOOOH YEAH!!!!!!
```

Comment:
Consider adding more exclamation points.

Reply:
The Luchador hasn't had enough coffee yet!